src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -748,20 +751,70 @@ const UsersConfigurationRow = ({
     );
 };
 
+const SshKeysRow = ({
+    id, item, onChange, idx, removeitem,
+}) => {
+    const [isLoading, setIsLoading] = useState(false);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1150-20230714-115437-e66ad82a-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)